src/index.html

```diff
@@ -1,13 +1,41 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="main-fhoto"
+      >
+      <span class="card__item">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </span>
+      <div class="card__code">Код товара: 195434</div>
+      <div class="card__mark">
+        <div class="stars" data-qa="stars">
```

Comment:
add modifier class (copy styles from the previous task 'stars', but only if it's approved)

```suggestion
        <div class="stars">
```

Reply:
It is copy of code from task with stars

Reply:
Oh, and this task was approved